queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] In order to get accurate query result, let us improve on the `date_of_birth` ` BETWEEN '2016-01-01' AND '2019/31/12'`

Reply:
Hi @addod19 👋 Kindly elaborate on the issue, do you mean that you want me to exclude the year 2016 and 2019 and query the results of 2017 and 2018?
![review](https://user-images.githubusercontent.com/30289772/236195551-a0beb5e4-155d-4246-af02-92c1386e1ee1.PNG)